### PR TITLE
Add usage instructions to README.

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,6 +17,15 @@ If you configure your local DNS server to include the Golink server as `go`, the
...
  name (FQDN) of the server.
 
+## Usage
+
+```bash
+python3 -m venv venv
+source venv/bin/activate
+pip3 install --upgrade pip
+pip3 install -r requirements.txt
+PYTHONPATH=`pwd`:$PYTHONPATH python3 golink/webapp.py --auth anonymous --database golinks.sqlite
```
Comment (dcoles) [inline README.md]: No need to set `PYTHONPATH`. You can run it directly with the `-m` option.

```suggestion
python3  -m golink.webapp --auth anonymous --database golinks.sqlite
```

**Review by dcoles — CHANGES_REQUESTED** [review]

File: README.md [hunk 2]
```diff
@@ -17,6 +17,15 @@ If you configure your local DNS server to include the Golink server as `go`, the
 [http://go/example](http://go/example) directly in your browser without needing to provide the fully qualified domain
  name (FQDN) of the server.
 
+## Usage
+
+```bash
+python3 -m venv venv
+source venv/bin/activate
+pip3 install --upgrade pip
```
Comment (dcoles) [inline README.md]: I'm not sure if upgrading `pip` is strictly necessary if you've just created a virtual-env. Was there a specific need for it?

Comment (ghost) [reply README.md]: Removed. No, it just happened that my system pip was out of date, but I don't think there's anything that requires an absolutely latest pip.

**Review by dcoles — APPROVED** [review]
Thank you for the PR. Updated documentation is always greatly appreciated.

**Pull request merged**